frontend-cert/css-projects/blog-post-card/user-stories.md

```diff
@@ -0,0 +1,20 @@
+1. You should have a `div` with a class of `blog-post-card` to hold all your card elements.
+
+2. You should have an image with a valid `alt` text and the class of `post-img`.
```

Comment:
Here we can provide a URL as we did for other labs. "For the `src` attribute of your image you can use `cover-photo.jpg` if you would like."